go.mod

```diff
@@ -1,71 +1,76 @@
 module github.com/rancher/terraform-provider-rancher2
 
-go 1.22.0
+go 1.23.4
 
-toolchain go1.22.5
+toolchain go1.23.6
 
 require (
 	github.com/ghodss/yaml v1.0.0
 	github.com/hashicorp/go-version v1.6.0
 	github.com/hashicorp/terraform-plugin-sdk v1.17.2
-	github.com/rancher/norman v0.0.0-20240708202514-a0127673d1b9
-	github.com/rancher/rancher v0.0.0-20240716141526-e0d2afd007d8
+	github.com/rancher/norman v0.5.2
+	github.com/rancher/rancher v0.0.0-20250227174106-7829dbe62d7f
 	github.com/rancher/rancher/pkg/apis v0.0.0
 	github.com/rancher/rancher/pkg/client v0.0.0
-	github.com/stretchr/testify v1.9.0
-	golang.org/x/crypto v0.25.0
-	golang.org/x/sync v0.7.0
+	github.com/stretchr/testify v1.10.0
+	golang.org/x/crypto v0.33.0
+	golang.org/x/sync v0.11.0
 	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api v0.30.2
-	k8s.io/apimachinery v0.30.2
-	k8s.io/apiserver v0.30.1
+	k8s.io/api v0.32.2
+	k8s.io/apimachinery v0.32.2
+	k8s.io/apiserver v0.32.2
 	k8s.io/client-go v12.0.0+incompatible
```

Comment:
Interestingly we are using such an old version of the package, can you give it a try to bump it to v0.32.2 ?

Reply:
`k8s.io/client-go` is later replaced with `k8s.io/client-go v0.32.2` at the bottom of the file. I'm unsure why a replace statement is used, but I kept it as is in case there is a functional reason for this.

Reply:
We usually put the replace directive at the top of the file, but that is not the case here, I guess that is why I missed it. :)